client/gutenberg/editor/index.js

```diff
@@ -19,6 +19,11 @@ export default function() {
 		page( '/gutenberg/post/:site?/:post?', siteSelection, post, makeLayout, clientRender );
 		page( '/gutenberg/page', siteSelection, sites, makeLayout, clientRender );
 		page( '/gutenberg/page/:site?/:post?', siteSelection, post, makeLayout, clientRender );
+
+		if ( config.isEnabled( 'manage/custom-post-types' ) ) {
+			page( '/gutenberg/edit/:type', siteSelection, sites, makeLayout, clientRender );
+			page( '/gutenberg/edit/:type/:site?/:post?', siteSelection, post, makeLayout, clientRender );
```

Comment:
using `:custom-post-type` here could possibly prevent a heap of confusion when viewed outside of this context